pkg/assets/handler.go

```diff
@@ -17,7 +17,7 @@ type RenderParams map[string]interface{}
 
 type RenderFunc func([]byte, RenderParams) ([]byte, error)
 
-type readerApplier interface {
-	Reader([]byte, RenderFunc, RenderParams)
-	Applier(ctx context.Context) error
+type resourceHandler interface {
```

Comment:
Renaming the members of this interface triggered a lot of other changes in this PR. What would the PR have looked like after adding a `Delete` method instead of changing the name?

Reply:
Cluster scoped resources, which are namespaces, cluster roles and cluster role bindings. All the others need not have a deleter for now.

Reply:
OK